solidity/contracts/hooks/ArbL2ToL1Hook.sol

```diff
@@ -81,7 +81,7 @@ contract ArbL2ToL1Hook is AbstractMessageIdAuthHook {
     ) internal override {
         bytes memory payload = abi.encodeCall(
             AbstractMessageIdAuthorizedIsm.verifyMessageId,
-            message.id()
+            (message.id(), metadata.msgValue(0))
```

Comment:
should we rename from `verifyMessageId` given its also verifying value?

Reply:
ok renamed it from verifyMessageId to preVerifyMessage